corehq/apps/users/views/mobile/users.py

```diff
@@ -618,9 +619,13 @@ def update_user_groups(request, domain, couch_user_id):
     form.fields['selected_ids'].choices = [(id, 'throwaway') for id in Group.ids_by_domain(domain)]
     if form.is_valid():
         user = CommCareUser.get(couch_user_id)
+        current_group_ids = user.get_group_ids()
```

Comment:
Slight preference for `old` instead of `current`, I think it's clearer.

Reply:
https://github.com/dimagi/commcare-hq/pull/30117/commits/15978fb67f23017f667a64e07b283913f14cc3e3